### PR TITLE
Fix PlayerAuthInput

### PR DESCRIPTION
**Pull request by Prim69**
i have no idea why this is a signed int what the fuck mojang

https://github.com/Sandertv/gophertunnel/blob/f8b092d0378cc7fdbf87640dae4def2d06fddeb4/minecraft/protocol/packet/player_auth_input.go#L172
https://github.com/CloudburstMC/Protocol/blob/cc778875eb5811dd836ad2d49b2a59446edcf473/bedrock/bedrock-v428/src/main/java/com/nukkitx/protocol/bedrock/v428/serializer/PlayerAuthInputSerializer_v428.java#L105

### CODE REVIEW

File: src/PlayerAuthInputPacket.php [hunk 1]
```diff
@@ -54,10 +54,10 @@ class PlayerAuthInputPacket extends DataPacket implements ServerboundPacket{
 	private ?Vector3 $vrGazeDirection = null;
 	private int $tick;
 	private Vector3 $delta;
-	public ?ItemInteractionData $itemInteractionData;
-	private ?ItemStackRequest $itemStackRequest;
+	public ?ItemInteractionData $itemInteractionData = null;
+	private ?ItemStackRequest $itemStackRequest = null;
 	/** @var PlayerBlockAction[]|null */
-	private ?array $blockActions;
+	private ?array $blockActions = null;
```
Comment (dktapps) [inline src/PlayerAuthInputPacket.php:60]: what does this have to do with it?

Comment (Prim69) [reply src/PlayerAuthInputPacket.php:60]: Nothing, it's just another issue I encountered, when var dumping packet->getInteractionData I would get "Typed property cannot be accessed before initialization"

Comment (Prim69) [reply src/PlayerAuthInputPacket.php:60]: Alvin mentioned checking flags, should I revert this?
